test/FibonacciBases_spec.js

```diff
@@ -0,0 +1,65 @@
+const FibonacciBase = require('../public/javascripts/FibonacciBase')
+const expect = require('chai').expect
+
+describe('FibonacciBase', function() {
+  let fibonacciInstance
+  beforeEach(function() {
+    fibonacciInstance = new FibonacciBase()
+  })
+  describe('#fibonacci(array)', function() {
+    it('should return an array of two numbers', function() {
+      let fibonacci = fibonacciInstance.fibonacci([])
+      expect(fibonacci).to.be.an('array')
+      expect(fibonacci).to.have.length.of.at.least(2)
+      expect(fibonacci).to.have.length.of.at.most(2)
+      expect(fibonacci[0]).to.be.a('number')
+      expect(fibonacci[1]).to.be.a('number')
+    })
+    it('should return an array containing second number given and the addition of the two inputs', function() {
+      let fibonacci = fibonacciInstance.fibonacci([2,3])
+      expect(fibonacci).to.eql([3,5])
+    })
+  })
+
+  describe('#decimalToFibonacci(decimal)', function() {
+    it('should take a decimal number and return a string - a binary based on the Fibonacci sequence', function() {
+      expect(fibonacciInstance.decimalToFibonacci(21)).to.eql('1100000')
+      expect(fibonacciInstance.decimalToFibonacci(36)).to.eql('100000100')
+      expect(fibonacciInstance.decimalToFibonacci(2)).to.eql('11')
+    })
+    it('should handle large numbers', function() {
+      expect(fibonacciInstance.decimalToFibonacci(239994848596)).to.eql('10000010000100101010010100000010010000101010000010000100')
```

Comment:
@rachel-ftw, I had to refactor to account for larger numbers. I was using modulus to trim the last digit off of the binary(the input was a number at this time) and I noticed errors when I tried a larger number. That is when i changed to this approach. Thanks for reviewing.